backend/src/v1/routes/wordRouter.ts

```diff
@@ -1,17 +1,35 @@
-import { Router } from "express";
-
-import { ProvideWordController, WordController } from "../controllers/wordController";
-
-export const WordRouter = Router();
-
-const controller: WordController = ProvideWordController();
-
-WordRouter.get("/", controller.getWords);
-
-WordRouter.get("/:id", controller.getWord);
-
-WordRouter.post("/", controller.addWord);
-
-WordRouter.put("/:id", controller.updateWord);
-
-WordRouter.delete("/:id", controller.deleteWord);
\ No newline at end of file
+import { Router } from 'express';
+import * as repository from '../data/repositories/wordRepository.js';
+
+const wordRouter = Router();
+
+wordRouter.get('/', async (req, res) => {
+	const languageCode = req.query.language_code as string;
+	const result = await repository.getWords(languageCode);
```

Comment:
I don't think this is going to do what you want. To make this return a 400 or some other error, you'll need a wrapper. I think this will get quietly dropped, at least from the API caller's perspective.

Reply:
yea i need to update the overall error handling for the API like you mentioned.